WorkloadManagementSystem/Client/CPUNormalization.py

```diff
@@ -123,9 +124,12 @@ def getCPUNormalization( reference = 'HS06', iterations = 1 ):
   except ( TypeError, ValueError ), x :
     return S_ERROR( x )
 
-  # This number of iterations corresponds to 250 HS06 seconds
+  # This number of iterations corresponds to 1kHS2k.seconds, i.e. 250 HS06 seconds
+  # 06.11.2015: fixed absolute normalization w.r.t. MJF at GRIDKA
+  from DIRAC.ConfigurationSystem.Client.Helpers.Operations import Operations
+  corr = Operations().getValue( 'JobScheduling/CPUNormalizationCorrection', 1. )
```

Comment:
The number we should put now is 0.6 as the Dirac absolute normalisation is 0.6 HS06
It has drifted since Ricardo made an absolute calibration as we changed OS, compiler, python version etc... 
I tuned it w.r.t. MJF assuming this one is correct...